backend/api/tests/handlers/dynamodb/put-service-client.test.ts

```diff
@@ -62,15 +68,24 @@ const expectedDynamoRecord = {
 describe("putServiceClientHandler tests", () => {
     beforeEach(() => {
         jest.clearAllMocks();
+        process.env = {
+            ...originalEnv,
+            IDENTITY_VERIFICATION_ENABLED: "Yes"
+        };
+    });
+
+    afterEach(() => {
+        process.env = originalEnv;
     });
 
     it("calls the dynamo client with a put command with the expected values and returns a 200 with the expected response body", async () => {
         const putItemSpy = jest.spyOn(DynamoDbClient.prototype, "put").mockResolvedValue({
             $metadata: {}
         });
 
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
Another `@ts-ignore` comment, I know the concept of invoking the lambda with a status code is odd, but that is the shape of the incoming data if you look at the step function in the console

Reply:
Thanks. Yes: this makes sense and I've worked through cleaning these up.